docs/blog/2019-07-03-themes-are-stable/index.md

```diff
@@ -0,0 +1,74 @@
+
+The answer is now. Like right now. If you're reading this post then themes are stable.
+
+# What is stable?
+
+The core theme APIs have been stable for a long time under the `__experimentalThemes` flag in `gatsby-config.js`. Since they're being used in production by a number of different companies to great effect, we're promoting these APIs, specifically composition and shadowing, to stable within Gatsby core so that people can take advantage of them with confidence.
+
+## A brief review of composition
+
+`gatsby-config` composition is the core abstraction behind how themes work. It means that you can write a `gatsby-config` in your plugin and it will be combined with any other plugin's `gatsby-config` and the site's `gatsby-config`. This results in a single combined `gatsby-config` that is then used to build your Gatsby project.
+
+Stability for composition means that any plugin can now take advantage of adding configuration, plugins, and metadata via a `gatsby-config`. The `gatsby-config` in any plugin can be an object, similar to the way any Gatsby project works today, or a function whose argument is the options specified by a user.
+
+```js:title=my-plugin/gatsby-config.js
+module.exports = options => {
+  return {
+    plugins: [],
+  }
+}
+```
+
+These options can be used to conditionally add plugins, mappings, or anything else you can do in a `gatsby-config.js`.
+
+## A brief review of shadowing
+
+Shadowing allows you to override specific components by creating a new file and exporting a component from that file. If the theme you're using has a `Header` component at `my-theme/src/components/header.js` then you can replace the component in your own site by creating a new file at `my-site/src/my-theme/components/header.js`.
+
+Stability for shadowing means that creating a single file is all you need to start changing how your theme renders your site. Additionally, we've added the ability to import and extend the parent component, allowing the use of props for customizing features of any React component.
+
+```js:title=my-site/src/my-theme/components/header.js
+import Header from "my-theme/src/components/header"
+
+export default props => <Header {...props} myProp={true} />
+```
+
+# What is different?
+
+The best API is no API, so we're getting rid of the `__experimentalThemes` key in `gatsby-config.js` and enabling any plugin to participate in composition or shadowing. Themes are now "part of how Gatsby works" instead of being a separate API. It looks like this:
+
+```js
+module.exports = {
+  plugins: [
+    {
+      resolve: "gatsby-theme-blog",
+      options: {},
+    },
+  ],
+}
+```
+
+note: `__experimentalThemes` is deprecated and we didn't remove it yet to allow time to migrate. Importantly there can be no `__experimentalThemes` usage if you want to use the new stable APIs in plugins. Migration is as easy as moving the theme declarations from `__experimentalThemes` to `plugins`.
```

Comment:
I think I know what this is saying, but the wording in the second sentence could be a little clearer – stating something like: `__experimentalThemes` will continue to work, but you should move themes into the `plugin` array for the new stable API